Amethyst/Events/HotKeyManager.swift

```diff
@@ -68,41 +69,47 @@ final class HotKeyManager: NSObject {
         }
     }
 
-    func setUpWithWindowManager(_ windowManager: WindowManager, configuration: UserConfiguration) {
+    func setUpWithWindowManager(_ windowManager: WindowManager<Application>, configuration: UserConfiguration) {
         constructCommandWithCommandKey(CommandKey.cycleLayoutForward.rawValue) {
-            windowManager.focusedScreenManager()?.cycleLayoutForward()
+            let screenManager: ScreenManager<Application.Window>? = windowManager.focusedScreenManager()
+            screenManager?.cycleLayoutForward()
         }
 
         constructCommandWithCommandKey(CommandKey.cycleLayoutBackward.rawValue) {
-            windowManager.focusedScreenManager()?.cycleLayoutBackward()
+            let screenManager: ScreenManager<Application.Window>? = windowManager.focusedScreenManager()
```

Comment:
This logic seems like it should be factored into its own function and replaced with a `guard` here